includes/external-connection-cpt.php

```diff
@@ -687,3 +724,73 @@ function filter_post_updated_messages( $messages ) {
 	return $messages;
 }
 
+/**
+ * Get the REST API root of the remote site.
+ *
+ * @since  1.6.0
+ *
+ * @param string $site_url Remote site URL.
+ *
+ * @return false|string
+ */
+function get_rest_url( $site_url ) {
+
+	$source = wp_remote_get( $site_url );
+
+	if ( ! is_wp_error( $source ) ) {
```

Comment:
@dinhtungdu Need to remove the negation (!) operator here, otherwise this will return false on success

Reply:
Oh silly me 😅. Thank you, Darin!